packages/color-block/src/components/cards/CardsItem.tsx

```diff
@@ -138,6 +139,7 @@ export const CardsItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onD
                                                 />
                                             }
                                             triggerElement={
+                                                // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions
```

Comment:
Same as above.

Reply:
Won't tackle as its an unreleased block and needs some work anyway before go live